src/compas/scene/context.py

```diff
@@ -26,6 +26,22 @@ def redraw():
 redraw.__pluggable__ = True
 
 
+@pluggable(category="drawing-utils")
+def predraw():
```

Comment:
Docstrings to explain what these do?

Reply:
@Licini and please add them to `__init__` and to the api docs.